views/group/new.pug

```diff
@@ -75,6 +82,9 @@ block scripts
   script(src='https://cdn.jsdelivr.net/npm/@fullcalendar/daygrid@4.3.0/main.min.js')
   script(src='https://cdn.jsdelivr.net/npm/@fullcalendar/timegrid@4.3.0/main.min.js')
   script(src="/js/picker.js")
+  if isEditing
+    script const isEditing = true
+    script const groupId = !{JSON.stringify(groupId)}
```

Comment:
Is this necessary?

Reply:
Both of the variables are used by group/create.js. But see my comment above, I'll see if there's solution. 😄

Reply:
So there's no way for pug template engine to pass variables to exterior javascript files, and the backend can't pass variables to those javascript files with res.render either, unless we integrate webpack (soon). So this simple redeclaring way is the nice way. Here's some [ref](https://riptutorial.com/pug/example/29560/value-interpolation-in-javascript-code). This is actually the easiest way, other ways include nasty packages.